components/text-utilities.html

```diff
@@ -0,0 +1,116 @@
+        <li><a class="nav-link link-active" href="#">Docs</a></li>
+        <li><a class="nav-link" href="#">Github</a></li>
+      </ul>
+    </nav>
+
+    <div class="page-content">
+      <aside class="side-navigation">
+        <div class="getting-started-link active-side-nav">
+          <a href="/getting-started.html">GETTING STARTED</a>
+        </div>
+        <div class="components-list-btn">
+          <a href="#">COMPONENTS</a>
+        </div>
+        <ul class="side-nav-items-list">
+          <li class="side-nav-item">
+            <a href="/components/avatar.html">Avatar</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/alert.html">Alert</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/badge.html">Badge</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/button.html">Button</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/card.html">Card</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/image.html">Image</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/input.html">Input</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/lists.html">Lists</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/modal.html">Modal</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/navigation.html">Navigation</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/rating.html">Rating</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/simple-grid.html">Simple Grid</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/text-utilities.html">Text Utilities</a>
+          </li>
+          <li class="side-nav-item">
+            <a href="/components/toast.html">Toast</a>
+          </li>
+        </ul>
+      </aside>
+      <main class="main-page-content">
+        <section class="section-content">
+          <h1>Text Utilities</h1>
+          <p>
+            Text utilities come handy when we want to alter the text sizes.
+            Below are some text sizes for heading.
+          </p>
+          <div class="text-container">
+            <h1 class="heading-xxl">This is the biggest heading.</h1>
+            <h2 class="heading-xl">This is the second biggest heading.</h2>
+            <h3 class="heading-lg">This is the third biggest heading</h3>
+            <h4 class="heading-md">This is the medium sized heading</h4>
+            <h5 class="heading-sm">This is the smaller heading</h5>
+            <h6 class="heading-xsm">This is the smallest heading</h6>
+          </div>
+
+          <pre class="component-code-container">
+          <code class="html">
```

Comment:
`.html` class is not defined in the  stylesheet

Reply:
In future I'll highlight this code using some library hence keep this class as of now, but if not needed I'll remove it.